backend/backend.go

```diff
@@ -73,13 +73,21 @@ var fixedURLWhitelist = []string{
 	"https://shiftcrypto.support/",
 	// Exchange rates.
 	"https://www.coingecko.com/",
-	// Block explorers.
+	// Block explorers (btc).
 	"https://blockstream.info/tx/",
 	"https://blockstream.info/testnet/tx/",
+	"https://mempool.space/tx/",
+	"https://mempool.space/testnet/tx/",
+	// Block explorers (ltc).
 	"https://sochain.com/tx/LTCTEST/",
 	"https://blockchair.com/litecoin/transaction/",
+	// Block explorers (eth).
 	"https://etherscan.io/tx/",
 	"https://goerli.etherscan.io/tx/",
+	"https://sepolia.etherscan.io/tx/",
+	"https://ethplorer.io/tx/",
+	"https://goerli.ethplorer.io/tx/",
+	"https://sepolia.ethplorer.io/tx/",
```

Comment:
instead of adding them all here, it's better to add another loop in SystemOpen() that goes through all explorers, so that one only has to modify the list of explorers when making changes (adding or removing explorers).